lib/tasks/update_search_tsvectors.rake

```diff
@@ -0,0 +1,65 @@
+namespace :search do
+
+  class VectorUpdater
+    attr_reader :redis, :start, :batch_size, :resource
+
+    def initialize(args)
+      @redis = Redis.new(url: ENV['REDIS_URL'], db: 15)
+      @resource = args[:resource]
+      @batch_size = args[:batch_size].to_i
+      @start = args[:start].to_i
+    end
+
+    def start_id
+      if redis.exists(rkey)
```

Comment:
Not blocking - we can't override the redis value via the args we pass in here. That's ok with me but maybe not expected

Reply:
I don't think I follow--we can manually change the value of the resource key if necessary, is that what you mean?

Reply:
Noting, this only applies if the redis key checkpoint value exists. 

If we pass args in and a redis key exists then redis key takes precedence. 

> we can manually change the value of the resource key if necessary, is that what you mean?
Yep but we can't do this via rake task args, just via the redis key (if it exists)

Reply:
Right. It had to be one of them that took precedence. If we need to start completely over we can just delete the key manually.